### PR TITLE
Week one slides

### PR DESCRIPTION
**Pull request by JasonGibson274**
I still need t check for typos but submitting for content checks.

### CODE REVIEW

**Comment by chsahit** [comment]
Would it be fair to assume they've made a Github account by this point?

**Comment by JasonGibson274** [comment]
Yes I will remove that section.

**Comment by chsahit** [comment]
Didn't see any typos...I'm approving but I'm going to ask Evan to add his slide

**Review by chsahit — APPROVED** [review]

File: slides/week1/week1.org [hunk 1]
```diff
@@ -76,7 +76,7 @@ briefly mention what each sensor is
 - IMU
 - encoders
 
-* IGVC Software
```
Comment (JasonGibson274) [inline slides/week1/week1.org]: *Software

File: slides/week1/week1.org [hunk 2]
```diff
@@ -156,24 +156,24 @@ the arrows are topics and the circles are nodes
  - On a Windows host, you may need to turn off Hyper-V as well.
 
 *** 1. Go to =File->Import Appliance=
-      [[file:https://i.imgur.com/keQmMy4.png]]
+	  [[file:https://i.imgur.com/keQmMy4.png]]
```
Comment (jgkamat) [inline slides/week1/week1.org]: I :heart: tabs, but you shouldn't be mixing tabs and spaces like this :)

**Pull request merged**